README.md

```diff
@@ -23,7 +23,7 @@ The possibilities are endless.
 Now that you have not done anything it's time to build your application:
 
 ```
-
+ 
```

Comment:
I think you just break the rule of "write nothing" due to introduced a space character; This should be removed.

Reply:
bad